sections/multicolumn.liquid

```diff
@@ -219,6 +219,34 @@
       "default": false,
       "label": "t:sections.multicolumn.settings.swipe_on_mobile.label"
     },
+    {
+      "type": "select",
+      "id": "color_scheme",
+      "options": [
+        {
+          "value": "accent-1",
+          "label": "t:sections.all.colors.accent_1.label"
+        },
+        {
+          "value": "accent-2",
+          "label": "t:sections.all.colors.accent_2.label"
+        },
+        {
+          "value": "background-1",
+          "label": "t:sections.all.colors.background_1.label"
+        },
+        {
+          "value": "background-2",
+          "label": "t:sections.all.colors.background_2.label"
+        },
+        {
+          "value": "inverse",
+          "label": "t:sections.all.colors.inverse.label"
+        }
+      ],
+      "default": "background-1",
+      "label": "t:sections.all.colors.label"
+    },
```

Comment:
Based on the issue I think it needs a container color scheme picker as well that would apply the color to the block 🤔

Reply:
[screenshot](https://screenshot.click/22-01-b9fcw-ozuhs.png)

Reply:
This also goes beyond what I'd consider to be a quick win.  @Oliviammarcello Can you see if we're okay with the current implementation?